pimd/pim_zlookup.c

```diff
@@ -520,17 +520,16 @@ int pim_zlookup_sg_statistics(struct channel_oil *c_oil)
 	pim_sgaddr sg;
 	int count = 0;
 	int ret;
+	pim_sgaddr more;
```

Comment:
Won't we have a Coverity warning here about an uninitialized variable?

Reply:
No it should not because it's not being used without being assigned a value in the function. I have initialised it now 👍